static/lang/re-en.json

```
@@ -3339,6 +3339,11 @@
                     },
                     "Single": "Single ikon with multiple copies."
                 },
+                "RedGoldMortality": {
+                    "DamageNote": "When the target would regain Hit Points, it must attempt a Will save against your class DC to determine the effects. @UUID[Compendium.pf2e.feat-effects.Item.TpcfUI2y6Shg63V0]{Effect: Red-Gold Mortality}",
```

```suggestion
                    "DamageNote": "If you successfully damage the enemy, when the target would regain Hit Points, it must attempt a Will save against your class DC to determine the effects. @UUID[Compendium.pf2e.feat-effects.Item.TpcfUI2y6Shg63V0]{Effect: Red-Gold Mortality}",
```
We actually do need the damage bit in there, as an enemy could be immune to the damage and then this wouldn't take effect.